<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>What is your Name ?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameInput"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>What are your favoirate languages?</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="javaCheckbox">
        <property name="text">
         <string>Java</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PhpCheckbox">
        <property name="text">
         <string>PHP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cppCheckbox">
        <property name="text">
         <string>C++</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="pythonCheckbox">
        <property name="text">
         <string>Python</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Where are you from?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="fromComboBox">
      <item>
       <property name="text">
        <string>Delhi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Grater Noida</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gaziabad</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Merrut</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Noida</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Saharanpur</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Ary You Happy?</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="HappyYesRB">
        <property name="text">
         <string>Yes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="HappyNoRB">
        <property name="text">
         <string>No</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="submitBtn">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="coloseBtn">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>coloseBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
